app/components/courses/international_students_component.html.erb

```diff
@@ -1,8 +1,16 @@
 <div class="govuk-!-margin-bottom-8">
-  <h2 class="govuk-heading-l" id="section-international-students">International students</h2>
+  <h2 class="govuk-heading-l" id="section-international-students">International candidates</h2>
   <div data-qa="course__international_students">
     <p class="govuk-body"><%= visa_sponsorship_status %></p>
-    <p class="govuk-body">You do not need a visa if you already have settled or pre-settled status under the EU Settlement Scheme, or if you’re an Irish citizen.</p>
-    <p class="govuk-body">Candidates with qualifications from non-UK institutions may need to provide evidence of comparability by applying for a <%= govuk_link_to("UK ENIC statement", "https://www.enic.org.uk") %>.</p>
+    <h3 class="govuk-heading-m">International qualifications</h3>
+    <p class="govuk-body">If your qualifications come from a non-UK institution, we may want to see a ‘statement of comparability’. This will show us how they compare to UK qualifications.</p>
```

Comment:
```suggestion
    <p class="govuk-body">If your qualifications come from a non-UK institution, we may want to see a ‘statement of comparability’ from UK ENIC. This will show us how they compare to UK qualifications.</p>
```